adafruit_requests.py

```diff
@@ -637,11 +589,9 @@ def _send_request(
         if json is not None:
             assert data is None
             # pylint: disable=import-outside-toplevel
-            try:
-                import json as json_module
-            except ImportError:
-                import ujson as json_module
-            data = json_module.dumps(json)
+            import json
+
+            data = json.dumps(json)
```

Comment:
This will fix the error detected by pytest.
```suggestion
            import json as json_module

            data = json_module.dumps(json)
```
Too many jsons in the kitchen !

Reply:
Thanks! I will finish this today (will be out this morning). Yes, we could just import it at the top. I'm not sure why there was a conditional import. Since it's built-in, the size is minimal.